TowersOfHanoi/TowersOfHanoi.cs

```diff
@@ -1,12 +1,199 @@
 ﻿using System;
+using System.Collections;
+using System.Collections.Generic;
 
-namespace TowersOfHanoi
-{
-    class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello World!");
+namespace TowersOfHanoi {
+    class Program {
+        static void Main (string[] args) {
+            Game towerOfHanoiGame = new Game ();
+            //enter game
+            Console.WriteLine ("---------------------------------");
+            Console.WriteLine ("    Let's Play Tower of Hanoi    ");
+            Console.WriteLine ("---------------------------------");
+            Console.WriteLine (" ");
+            //create a loop to keep playing
+            towerOfHanoiGame.Play ();
+            towerOfHanoiGame.PrintBoard ();
         }
     }
-}
+    /*
+    Blocks are weighted by number, the greater number = more weight.
+    Blocks are interacted with by the user, moved between towers.
+    Blocks need to be linked to a tower. 4 needs to be the first block in so 1 can be moved first.
+     */
+    class Blocks {
+        public int weightedBlock { get; set; }
+        public Blocks (int weightedBlock) {
+            this.weightedBlock = weightedBlock;
+        }
+    }
+    /*
+    Towers hold blocks, initially Tower A holds all blocks.
+    User then moves blocks between 3 towers.
+     */
+    class Towers {
+        public Stack<Blocks> towerBlocks { get; set; }
+        public Towers () {
+            this.towerBlocks = new Stack<Blocks> ();
+        }
+
+    }
+    /*
+    Creates a game,
+    draws the board,
+    Allows the user to move blocks
+    Allows the user to choose towers.
+     */
+    class Game {
+        //variables &&|| definitions
+        Dictionary<String, Towers> MakeTowers = new Dictionary<String, Towers> ();
+        bool win = false;
```

Comment:
Instead of having all these variables at such a high scope on the class, it would be better if they were only within the method.